src/Behavioral.Automation/FluentAssertions/Abstractions/IAssertionBuilderWithInversion.cs

```diff
@@ -0,0 +1,13 @@
+using Behavioral.Automation.Elements;
+using System;
+
+namespace Behavioral.Automation.FluentAssertions.Abstractions
+{
+    public interface IAssertionBuilderWithInversion
+    {
+        IAssertionBuilderWithValidatedAssertion Be<TValue>(Func<IWebElementWrapper, TValue> valueAcessor, Func<TValue, TValue, bool> comparer, TValue value, string message);
+        IAssertionBuilderWithValidatedAssertion Be<T>(AssertionObject<T> assertion);
+        IAssertionBuilderWithValidatedAssertion Become<TValue>(Func<IWebElementWrapper, TValue> valueAcessor, Func<TValue, TValue, bool> comparer, TValue value, string message);
```

Comment:
I would use IEqualityComparer<TValue> type instead of Func<TValue, TValue, bool> comparer

Reply:
I'm actually against such approache because it's not verbose enough. Tou have to navigate to this type to understand, what you should pass there. 
On the other hand using of Func<> types is clear enought to understand, what type of parameter should you pass.